src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -175,7 +175,10 @@ trait BackendCall {
       else
         Future.successful(Option(dockerImage))
 
-    call.task.runtimeAttributes.docker map hashDockerImage getOrElse Future.successful(None) map hashGivenDockerHash
+    if (workflowDescriptor.configCallCaching)
```

Comment:
That might require a fair bit of plumbing to get the descriptor all the way down there